### PR TITLE
Traveller's rest and prey pod fixes

### PR DESCRIPTION
**Pull request by Profakos**
## About The Pull Request

- Traveller's rest had two airlock cyclelinks that did not work, as one of them was on a windoor, so I have removed them. Not playerfacing.
- The mob spawner spawned without clothes and runtiming, as instead of an outfit, it had the roboticist's jumpsuit added. I have made a new outfit and used that in a new spawner. Might be too specific of a spawner though.
- I have also added a dedicated roboticist spawner, which uses a full outfit. I originally wanted to use this one, but I realized, making this spawn with robotics ID and gear is a pretty big balance change, so I elected not to use that, but I have added it anyways, as it should be useful in the future (and I have already made the sprite).

### Mapping March

Ckey to receive rewards: N/A

## Why It's Good For The Game

Less warnings during initialization.

## Changelog

:cl:
fix: the roboticist in the prey pod now has clothes as intended
/:cl:

### CODE REVIEW

File: code/modules/mob_spawn/corpses/job_corpses.dm [hunk 1]
```diff
@@ -59,6 +59,11 @@
 	outfit = /datum/outfit/job/assistant
 	icon_state = "corpsegreytider"
 
+/obj/effect/mob_spawn/corpse/human/roboticist
```
Comment (san7890) [inline code/modules/mob_spawn/corpses/job_corpses.dm:62]: move this somewhere else so it isn't sandwiched between the Assistant type declaration and its initialize proc

File: code/modules/mob_spawn/corpses/job_corpses.dm [hunk 2]
```diff
@@ -59,6 +59,11 @@
...
+	name = "Roboticist"
```
Comment (san7890) [inline code/modules/mob_spawn/corpses/job_corpses.dm:63]:
```suggestion
	name = JOB_ROBOTICIST
```

File: code/modules/mob_spawn/corpses/mob_corpses.dm [hunk 3]
```diff
@@ -317,3 +317,11 @@
 	facial_hairstyle = "Shaved"
 	skin_tone = "caucasian1"
 	outfit = /datum/outfit/job/assistant/consistent
+
```
Comment (san7890) [inline code/modules/mob_spawn/corpses/mob_corpses.dm:320]:
```suggestion
```

only one new line is really needed here

**Review by san7890 — CHANGES_REQUESTED** [review]

File: code/modules/mob_spawn/corpses/job_corpses.dm (hunk 1 above)
Comment (Profakos) [reply code/modules/mob_spawn/corpses/job_corpses.dm:62]: Aw dang I was blind... sorry!

File: code/modules/mob_spawn/corpses/job_corpses.dm (hunk 2 above)
Comment (Profakos) [reply code/modules/mob_spawn/corpses/job_corpses.dm:63]: While I am here, I'll do the same with the other jobs too

**Comment by LemonInTheDark** [comment]
image
Was it intentional to spawn a human in the wall here?

**Comment by Profakos** [comment]
No, it wasn't, I am sorry.

**Comment by LemonInTheDark** [comment]
I will never forgive you for this crime (don't worry bout it, just thought you might want to know. noticed when doing lighting work)

**Pull request merged**